### PR TITLE
move to new android binding for plugin

### PR DESCRIPTION
**Pull request by remonh87**

### CODE REVIEW

File: android/src/main/kotlin/com/signify/hue/flutterreactiveble/PluginController.kt [hunk 1]
```diff
@@ -77,7 +78,7 @@ class PluginController {
         https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling
         */
         RxJavaPlugins.setErrorHandler { throwable ->
-            if (throwable is UndeliverableException && throwable.cause is BleException) {
+            if (throwable is UndeliverableException && (throwable.cause is BleException) || throwable.cause is RuntimeException) {
```
Comment (werediver) [inline android/src/main/kotlin/com/signify/hue/flutterreactiveble/PluginController.kt]: I don't think parentheses around `throwable.cause is BleException` are required.

What's the rationale behind adding `throwable.cause is RuntimeException`?

Do you mean what is written? The grouping is as follows: `(throwable is UndeliverableException && throwable.cause is BleException) || throwable.cause is RuntimeException)`.

Comment (remonh87) [reply android/src/main/kotlin/com/signify/hue/flutterreactiveble/PluginController.kt]: whoops indeed this will provide nasty side effects s oremoved it

**Comment by remonh87** [comment]
Fixes #16

**Comment by werediver** [comment]
As we concluded off-line, it looks like a good idea to _stop_ setting the global RX error handler in our plug-in. Let's track it in a separate issue.

**Comment by remonh87** [comment]
You are right created #18

**Pull request merged**